apps/web/pages/signup.tsx

```diff
@@ -143,7 +142,7 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
   // no token given, treat as a normal signup without verification token
   if (!token) {
     return {
-      props,
+      props: JSON.parse(JSON.stringify(props)),
```

Comment:
It was throwing error getServerSideProps cannot be serialized as JSON when we return only props.